### DISCUSSION TITLE
Download->edit -> upload file

### DISCUSSION THREAD

**Discussion by rsacher1** [SafeExamBrowser/seb-win-refactoring#532] [Q&A]
Using Moodle, students log in, retrieve their test-assignments as a word-file.
They need to start Word, and find that file, edit it and save it.
Then they need to browse for that file in Moodle and upload it.

Problem: After downloading into "Downloads", we cannot open it in Word ("access denied")

Problem 2: IF we were able to open that file from Downloads, HOW could we be sure, no other previously saved files can be opened for cheating?

**Comment by dbuechel** [comment]
Not being able to open the file from the downloads folder appears to be a configuration / access restriction issue on your machine. And unfortunately, you cannot prevent the other problem, unless you somehow supervise how and which file the students open and/or upload.
